resources/build/build-utils.sh

```diff
@@ -141,12 +141,41 @@ function printVersionUtilsDebug() {
     echo "VERSION_WITH_TAG: $VERSION_WITH_TAG"
 }
 
+function findShouldSentryRelease() {
+    # Default, in case $VERSION_ENVIRONMENT is improperly specified.
+    UPLOAD_SENTRY=false
+
+    # Dynamically configure whether or not to upload Sentry symbols.
+    case $VERSION_ENVIRONMENT in
+    # Actual release tiers
+    alpha)
+        UPLOAD_SENTRY=true
+        ;;
+    beta)
+        UPLOAD_SENTRY=true
+        ;;
+    stable)
+        UPLOAD_SENTRY=true
+        ;;
+
+    # Development builds; do not release (by default).
+    local)
+        UPLOAD_SENTRY=false
+        ;;
+    test)
+        UPLOAD_SENTRY=false
+        ;;
```

Comment:
These aren't actually needed. Suggest we note the development build cases in comment per suggestion on line 145.